arionBackend/tests/api/test_hierarchy.py

```diff
@@ -0,0 +1,33 @@
+"""
+This module holds the test cases for the different views.
+"""
+
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+
+class ViewsTestCase(APITestCase):
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a docstring to this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.tommartensen.de/coding_rules#rule_key=python%3AS1720)